index.html

```diff
@@ -1,20 +1,101 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Here we import the Boostrap CSS stylesheet -->
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
-    <!-- Add a link to your custom CSS here -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@400;500&family=Euphoria+Script&family=Redressed&family=Satisfy&display=swap"
+    rel="stylesheet">
+    <link rel="stylesheet" href="css/main.css">
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <section class="container">
+        <header class="row">
+            <div class ="col-sm-1" >
+                <img class="blue" src="./img/blue.jpg">
+            </div>
+            <div class="col-7">
+                <p class="header-text"><span>Maya Nelson</span> / Project Manager</p>
+            </div>
```

Comment:
combine these so the blue square is inside the same `col-` class as the header-text